### PR TITLE
ivy.lgamma

### LINKED ISSUE #19152: lgamma

### PR DESCRIPTION
**Pull request by marccgrau**
Close #19152

### CODE REVIEW

File: ivy/functional/ivy/experimental/elementwise.py [hunk 1]
```diff
@@ -21,6 +21,62 @@
...
+    """
+    return ivy.current_backend(x).lgamma(x, out=out)
+
+
+@handle_exceptions
+@handle_nestable
+@handle_array_like_without_promotion
+@handle_out_argument
+@to_native_arrays_and_back
+@integer_arrays_to_float
```
[the remark this answers is not in the document]
Comment (marccgrau) [reply ivy/functional/ivy/experimental/elementwise.py:78]: Apparently the decorator has been removed six days ago in #19205. So no need to include it anymore.

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py [hunk 2]
```diff
@@ -9,6 +9,35 @@
...
+    test_gradients=st.just(False),
+)
+def test_lgamma(
+    *,
+    dtype_and_x,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py:29]: You can remove this argument

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py [hunk 3]
```diff
@@ -9,6 +9,35 @@
...
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_function(
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        ground_truth_backend=ground_truth_backend,
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py:35]: remove this as well

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py [hunk 4]
```diff
@@ -9,6 +9,35 @@
 
 # Helpers #
 # ------- #
+# lgamma
+@handle_test(
+    fn_tree="functional.ivy.experimental.lgamma",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py:16]: use `"float"` instead of `"valid"` as dtypes

**Review by sherry30 — CHANGES_REQUESTED** [review]
Requested a few changes.

Can you add unsypported dtypes accordingly.
I think paddle doesn't support `float16` the rest you can test and find out.

Thanks

**Review by sherry30 — APPROVED** [review]
lgtm!
Thanks for the changes.

**Pull request merged**